test/intl402/DateTimeFormat/prototype/format/temporal-objects-formatting.js

```diff
@@ -0,0 +1,188 @@
+// Copyright (C) 2025 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-datetime-format-functions
+description: Different combinations of style options and Temporal types format correctly.
+locale: [en]
```

Comment:
I believe this'd need to be
```suggestion
locale: [en-US]
```

Reply:
I think this one got missed in the rewrite.